search-app/src/cmr/search/results_handlers/umm_json_results_handler.clj

```diff
@@ -2,6 +2,8 @@
   "Handles Collection umm-json results format and related functions"
   (:require
    [cheshire.core :as json]
+   [clojure.edn :as edn]
+   [clojure.string :as str]
```

Comment:
Delete both the edn and str namespace aliases, as they are not used in the file.